cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -49,3 +49,10 @@ def test_myservice_crud(setup_provider, context, order_catalog_item_in_ops_ui):
         myservice = MyService(appliance, service_name)
         myservice.update({'name': '{}_edited'.format(service_name)})
         # TODO - add rest of myservice crud like delete in next phase.
+
+
+@pytest.mark.uncollectif(lambda: current_version() < '5.8')
+@pytest.mark.parametrize('context', [ViaSSUI])
+@pytest.mark.parametrize('order_catalog_item_in_ops_ui', [['console_test']], indirect=True)
+def test_vm_console(setup_provider, context, configure_websocket, order_catalog_item_in_ops_ui):
+    pass
```

Comment:
Consider omitting the the placeholder test altogether and add working test in other PR.

Reply:
As we discussed on IRC, my other PR that is in 1LP currently adds the code to text: https://github.com/ManageIQ/integration_tests/pull/5529/files#diff-2e948dd0c6d3ef7b83e2dbc46c8e89caR54